modules/ROOT/pages/contribute.adoc

```diff
@@ -7,15 +7,12 @@ Many of our documentation repositories are open and we welcome your contribution
 
 To contribute to the English version of this documentation:
 
-. Select *Edit in Github* on the page you want to modify.
+. Sign the https://cla.salesforce.com/sign-cla[Salesforce.com Contributor License Agreement (CLA)] if this is your first time contributing. The CLA only needs to be signed once per GitHub account.
```

Comment:
I think the sentence structure might be better if you switch the order of the clauses; "If this is your first time ..., sign the ..."

Reply:
I agree, feels weird.